### ISSUE TITLE
STL file loads and slices but it wont save on SD card

### ISSUE THREAD

**Issue by fjimmy0** [Ultimaker/Cura#4252]
The program works just fine then when i try to save it to the SD card, an error occurs. Now i cant print anything.
![image](https://user-images.githubusercontent.com/42478716/44286074-6534fe80-a22d-11e8-9949-fe51c634645d.png)

![image](https://user-images.githubusercontent.com/42478716/44286123-97def700-a22d-11e8-81f2-dc66cf2859e2.png)
How do i fix this? Should i reinstall the program?

**Comment by LipuFei** [comment]
Hi @fjimmy0 Could you upload file? You can find it in
```
C:\Users\<your-user-name>\AppData\Roaming\cura\<cura-version>\cura.log
```

And, if you try to save it to a local hard drive, does it work?

**Comment by LipuFei** [comment]
Could be related to https://github.com/Ultimaker/Cura/issues/4495

**Comment by LipuFei** [comment]
Hi @fjimmy0 We have a candidate build for Cura 3.5.1, could you try it? We have a fix for #4495 in it and I wonder if it works in your case. Thanks!
Link: https://we.tl/t-0gqXweB9KG

**Comment by LipuFei** [comment]
I will close this issue because it's probably the same as #4495. That has been fixed for the next release 3.5.1. You can reopen it in case it's not the same issue.
